CHANGELOG.md

```diff
@@ -126,6 +126,7 @@
 - `tracegen`: Add option to pass in custom headers to export calls via command line (#7308)
 - `tracegen`: Provide official container images (#7179)
 - `scrapertest`: Add comparison function for pdata.Metrics (#7400)
+- `hostmetricsreceiver`: Add cpu.utilization metrics to cpu scrapper (#6221)
```

Comment:
Please move it to unreleased section

Reply:
thanks for the review @dmitryax. Changes pushed.